community/code-of-conduct.md

```diff
@@ -0,0 +1,166 @@
+# [Name Option 3] Workplace Harassment - An informal guide
+
+## Some basic fundamentals
+
+1. Humans are flawed and make mistakes on a regular basis while discovering new ways to hurt new people.
+
+2. We don't live in an ideal world, so we are not dealing with ideal scenarios.
+
+3. But the above statement doesn't provide wind to the practicality argument. Practicality argument is an exceptionally dangerous proposition in itself. So any time any particular statement is backed up by "being practical", it should take care of few things :-
+   1. Being practical means what can be done or is possible in some finite duration. So it should be aggressively progressive in nature and it shouldn't settle on "n" but "n + ∞". We can start from "n" but the vision should be clear to reach "n + ∞" as fast as possible.
+   2. It should be “enabler” in nature.
+
+4. Any democratic argument in case of harassment should be thrown out of the window immediately. For example, "9 out of 10 people are okay with this".
+
+5. Humans are not rational, so don't expect them to behave rationally. We are totally full of feelings.
+
+6. We don't accept snowflake argument. Some people have thick skin and are borderline stoic, and they think that "I am responsible for my feelings" and on the opposite end of emotional spectrum, are people who think that "You don't get to decide if you hurt someone". We will try to have an inclusive space for both of these people.
+
+7. Humans hate to forgive people if forgiving is tough.
+
+8. Convincing a human that they are wrong or have done wrong is exceptionally tough.
+
+9. Humans have double standards. Deal with it.
+
+## Basic Protocol in case you are harassed/bullied/found something offensive/weird/unacceptable
+
+Subject -> Who has been harassed/bullied
+
+Type 1 :- Constitutionally Legal Offense
+
+1. In this case the team is requested not to micromanage. let it go through official channels of college and judiciary and let them deal with it.
+
+Type 2 :- Everything Else
+
+1. If it has happened for the first time, dm the person. And let {pronoun} know that this behaviour is offensive to the subject. If not comfortable confronting then **????????????? @team comments**
+2. Post on a public forum that the subject finds this particular behaviour offensive/xyz. Name calling doesn't help.
+3. If it's repetitive, **what to do ???????????????????? @team , comments**
+
+## Basic Protocol in case you are accused of harassment/bullying/offending/doing weird shits/unacceptable behaviour
+
+Subject -> Who has been accused
+
+1. Well the subject can start by saying sorry, and ensuring that similar behaviour is not repeated in front of the person who has accused the subject "ever". If it's ingrained in the subject’s habit let the accussee know that, it's in your habit you will make sincere effort to decrease the frequency. The subject can take a note and share updates of progress {pronoun} have made on that person's feedback.
```

Comment:
Such things are extremely contextual. There could have been a big misunderstand due to which the other person felt uncomfortable. We have to think about this.

Reply:
We can add a `0` step - The subject should accept the behaviour (or the mistake) and make others aware of it. While the intention is not to shame the individual, I feel sorry without realizing what went wrong is more harmful. If the individual was mistaken, they should not be taking it on their ego, sharing what happened (at least with a closed group involving the accuser), and elaborating what amendments they'll make.

Reply:
@xypnox 
> IMO decrease the frequency is wrong. Stopping it altogether is better. We are not talking about smoking addiction here.
  
There can be things in habits like abusing(extreme levels :) ) , making sexual comments, which might just not vanish on asking and come up in conversation here and there.